.editorconfig

```
@@ -8,6 +8,6 @@ insert_final_newline=true
 indent_size=unset
 indent_style=space
 
-[*.{json,yml}]
+[*.{js,css,json,yml,pug}]
 indent_size=2
 indent_style=space
```

This looks good to me now. Before we merge this, should we just set this configuration for `[*]`? If not, I suggest we change this to:

```suggestion
[*.{css,js,json,pug,yml}]
indent_size=2
indent_style=space
```

---

> Before we merge this, should we just set this configuration for `[*]`?

Yes :+1: